pkg/operator/operator.go

```diff
@@ -87,8 +86,7 @@ func NewOpenShiftControllerManagerOperator(
 func (c OpenShiftControllerManagerOperator) getOperatorConfig() (
 	*operatorapiv1.OpenShiftControllerManager, error,
 ) {
-	return c.operatorConfigClient.OpenShiftControllerManagers().
-		Get(context.TODO(), "cluster", metav1.GetOptions{})
+	return c.operatorConfigLister.Get("cluster")
```

Comment:
the only actual change in an innocently looking commit.

Reply:
It does deserve a highlight on the PR description, sorry. It end up being part of this PR via the review process, on @gabemontero's comment.